### PR TITLE
Render missing security history events

### PR DESCRIPTION
**Pull request by jzohrab**
Addresses #13934 

I've added what I felt was the relevant detail for each event.

### CODE REVIEW

File: warehouse/templates/manage/account.html [hunk 1]
```diff
@@ -713,6 +713,34 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
         {% trans href=request.route_path('organizations.profile', organization=event.additional.organization_name), organization_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}Invitation to join <a href="{{ href }}">{{ organization_name }}</a> expired{% endtrans %}
       </strong>
 
+
+      {% elif event.tag == EventTag.Account.OrganizationRoleAdd %}
+      <strong>{% trans %}OrganizationRoleAdd{% endtrans %}</strong>
```
Comment (jzohrab) [inline warehouse/templates/manage/account.html:718]: I'm not sure what extra info to add, given inconsistent additional data, ref lines:

https://github.com/pypi/warehouse/blob/main/warehouse/organizations/services.py#L195-L214

File: warehouse/templates/manage/account.html [hunk 2]
```diff
@@ -713,6 +713,34 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
...
+      {% elif event.tag == EventTag.Account.OrganizationRoleChange %}
+      <strong>{% trans %}OrganizationRoleChange{% endtrans %}</strong>
```
Comment (jzohrab) [inline warehouse/templates/manage/account.html:720]: Inconsistent additional data, ref https://github.com/pypi/warehouse/blob/main/warehouse/manage/views/organizations.py#L1192-L1209

File: warehouse/templates/manage/account.html [hunk 3]
```diff
@@ -713,6 +713,34 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
...
+      {% elif event.tag == EventTag.Account.OrganizationRoleRemove %}
+      <strong>{% trans %}OrganizationRoleRemove{% endtrans %}</strong>
```
Comment (jzohrab) [inline warehouse/templates/manage/account.html:722]: Same, ref https://github.com/pypi/warehouse/blob/main/warehouse/manage/views/organizations.py#L754-L771

File: warehouse/templates/manage/account.html (hunk 1 above)
Comment (jzohrab) [reply warehouse/templates/manage/account.html:718]: My mistake, these are different constants :-), deleting the comments!

File: warehouse/templates/manage/account.html [hunk 4]
```diff
@@ -713,6 +731,34 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
...
+
+      {% elif event.tag == EventTag.Account.OrganizationRoleAdd %}
+      <strong>
+        {% trans href=request.route_path('organizations.profile', organization=event.additional.organization_name), organization_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}Role in <a href="{{ href }}">{{ organization_name }}</a> added{% endtrans %}
+      </strong>
+      {% elif event.tag == EventTag.Account.OrganizationRoleChange %}
+      <strong>
+        {% trans href=request.route_path('organizations.profile', organization=event.additional.organization_name), organization_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}Role in <a href="{{ href }}">{{ organization_name }}</a> changed{% endtrans %}
+      </strong>
+      {# TODO - determine what to add.  Ref https://github.com/pypi/warehouse/blob/main/warehouse/manage/views/organizations.py#L763-L771 #}
```
Comment (jzohrab) [inline warehouse/templates/manage/account.html:743]: TODO: Not sure what to add.

File: warehouse/templates/manage/account.html [hunk 5]
```diff
@@ -504,6 +504,20 @@ <h2>{% trans %}API tokens{% endtrans %}</h2>
   <section id="account-events">
     <h2>{% trans %}Security history{% endtrans %}</h2>
 
+    {% set role_strings = {
```
Comment (jzohrab) [inline warehouse/templates/manage/account.html:507]: I think this method is clean but it deviates from existing code, LMK if you don't like it.

**Comment by jzohrab** [comment]
Hm, well, apparently the `trans` tag doesn't like this line:

`<strong>{% trans %}{{ role_strings[event.tag] }}{% endtrans %}</strong>`

... so I'm going to infer that I can create a temporary variable similar to lines like this:

```
{% trans href=request.route_path('organizations.profile', organization=event.additional.organization_name), organization_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}Invited to join <a href="{{ href }}">{{ organization_name }}</a>{% endtrans %}
```

Sorry to abuse the CI system, I can't build and serve locally as mentioned above.

**Comment by jzohrab** [comment]
Fixed the build -- rookie mistake, I thought it was a code/build dependency problem somehow, was just docker fussing.  What a terrible hacker this PR makes me appear to be.  But it's true I guess. `<single_tear />`

**Comment by jzohrab** [comment]
CI failing:

```
754:9	Parse error: expected one of 'autoescape', 'block', 'blocktrans', 'comment', 'filter', 'for', 'if', 'ifchanged', 'ifequal', 'ifnotequal', 'not an intermediate Jinja tag name', 'spaceless', 'trans', 'verbatim', 'with' at 754:9	parse_error
```

hoping to get some guidance, as the jinja looks ok to me:

```
      {% elif event.tag in role_strings %}
      <strong>{% trans heading=role_strings[event.tag] %}{{ heading }}{% endtrans %}
      <small>
        {% trans project_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}
        Project: {{ project_name }}; role: {{ role_name }}
        {% endtrans %}
      </small>
```

File: warehouse/templates/manage/account.html [hunk 6]
```diff
@@ -713,6 +731,35 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
...
+      {% elif event.tag == EventTag.Account.OrganizationRoleChange %}
+      <strong>
+        {% trans href=request.route_path('organizations.profile', organization=event.additional.organization_name), organization_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}Role in <a href="{{ href }}">{{ organization_name }}</a> changed{% endtrans %}
+      </strong>
+      {# TODO - determine what to add.  Ref https://github.com/pypi/warehouse/blob/main/warehouse/manage/views/organizations.py#L763-L771 #}
+      {% elif event.tag == EventTag.Account.OrganizationRoleRemove %}
+      <strong>{% trans %}OrganizationRoleRemove{% endtrans %}</strong>
+
+      {% elif event.tag in role_strings %}
+      <strong>{% trans heading=role_strings[event.tag] %}{{ heading }}{% endtrans %}
```
Comment (di) [inline warehouse/templates/manage/account.html:748]:
```suggestion
      <strong>{% trans heading=role_strings[event.tag] %}{{ heading }}{% endtrans %}</strong>
```

File: warehouse/templates/manage/account.html [hunk 7]
```diff
@@ -713,6 +731,35 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
...
+      <small>
+        {% trans project_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}
+        Project: {{ project_name }}; role: {{ role_name }}
+        {% endtrans %}
+      </small>
+
+      {% elif event.tag in team_role_strings %}
+      <strong>{% trans heading=team_role_strings[event.tag] %}{{ heading }}{% endtrans %}
```
Comment (di) [inline warehouse/templates/manage/account.html:756]:
```suggestion
      <strong>{% trans heading=team_role_strings[event.tag] %}{{ heading }}{% endtrans %}</strong>
```

File: warehouse/templates/manage/account.html (hunk 6 above)
Comment (jzohrab) [reply warehouse/templates/manage/account.html]: Thank you, fixed.

File: warehouse/templates/manage/account.html (hunk 4 above)
Comment (jzohrab) [reply warehouse/templates/manage/account.html]: Added what was available.

**Comment by jzohrab** [comment]
Thanks @di for the comment.  I've completed this, I think, and removed all of the cruft comments from the PR.  CI now passes.

Can tweak as needed, and rebase when done, there are a few cruft comments.

**Review by di — APPROVED** [review]

**Pull request merged**